spinta/utils/scopes.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
+import re
+
+scope_re = re.compile(r'[^a-z0-9]+', flags=re.IGNORECASE)
+
+
+def name_to_scope(template: str, name: str, *, maxlen: int = None, params: dict = None):
+    scope = template.format(name=name, **params)
```

Comment:
It would be great to add a little comment what this function does and document that it shortens long scope names with a hash. 

Personally, reading code when there's a bunch of (even trivial) math/hashing operations with arbitrary integers is not easy and takes too much mental overhead.